locale/pt-br/knowledge/javascript-conventions/what-is-json.md

```diff
@@ -9,6 +9,11 @@ difficulty: 5
 layout: knowledge-post.hbs
 ---
 
+This page is not actively maintained and may be outdated. Please refer to [MDN's JSON page](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/JSON) instead.
```

Comment:
@nodejs/nodejs-pt If someone can offer a translation for these two sentences, that would be great.

Reply:
Thank you very much!